### PR TITLE
Chore: Added RPC for Arbitrum and updated Gnosis to point to Arbitrum Safe API

### LINKED ISSUE #242: Chore: update gnosis.js to support Arbitrum Safe Transaction API

### PR DESCRIPTION
**Pull request by codebuster22**
closes #242

### CODE REVIEW

**Review by fabianschu — APPROVED** [review]

File: tasks/utils/gnosis.js [hunk 1]
```diff
@@ -80,6 +82,7 @@ const getDelegates = async (safe, url) =>
 const api = (safe, network) => {
   const url = getUrl(network);
   return {
+    endpoint: url,
```
Comment (Zitzak) [inline tasks/utils/gnosis.js:85]: Why did you add this line, if it wasn't in the last iteration? Just for my own understanding

**Review by Zitzak — APPROVED** [review]

File: tasks/utils/gnosis.js (hunk 1 above)
Comment (codebuster22) [reply tasks/utils/gnosis.js:85]: there was no way to get the API url, before this, now using this property the user can get the endpoint.
